receiver/postgresqlreceiver/testdata/integration/02-create-extension.sh

```diff
@@ -11,4 +11,5 @@ EOSQL
 
 psql -v ON_ERROR_STOP=1 --username "root" --dbname "otel2" <<-EOSQL
   CREATE EXTENSION IF NOT EXISTS pg_stat_statements;
+  GRANT SELECT ON test2 TO otelu
```

Comment:
is this expected change?

Reply:
yes, explain need to have select permission